spec/prog/vnet/update_firewall_rules_spec.rb

```diff
@@ -132,15 +138,15 @@
 
   }
 
-  set private_ipv4_ips {
+  set private_ipv4_cidrs {
     type ipv4_addr;
     flags interval;
     elements = {
       10.0.0.0/26
     }
   }
 
-  set private_ipv6_ips {
+  set private_ipv6_cidrs {
```

Comment:
any scary backwards compatibility concerns here, for systems that have old set names laying around? Maybe they just live with them and it's not referred to by anything until they drop off? acceptable at our current stage.

Reply:
Not really, the fw_table is competely dropped in `update_firewall_rules` prog. So, the old set will stay there in the old system until a firewall rule update happens, until then, the set will only be referred by the nftables chain, locally, in the VM. I don't see an issue here